modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling sizes of 1, 50, and 400.
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Total average resource usage
+[options="header"]
+|===
+| Sample size | Parameters | Small (25 nodes)  | Medium (65 nodes) ^[2]^ | Large (120 nodes) ^[2]^
```

Comment:
I think we need to use different headers for the columns. The main driver is not the size of the cluster, but it's the test being run. So it should be perhaps something like _Test 1_, _Test 2_, _Test 3_, with the cluster size in parenthesis. I'm insisting on this because otherwise, the numbers sometimes don't make sense. For instance, with sampling 400 we see that netobserv uses more CPU than on a medium cluster. That sounds wrong, but it's just that the test scenario is completely different.

Reply:
cc @memodi

Reply:
Updated the headers to _Test 1, Test 2, Test 3_ along with the corresponding cluster sizes in parentheses.

Reply:
@jotak @mpandya15 - I am fine with having as Test 1, 2, 3 headers , I thinking that may be too abstract though, do we need to provide some visibility as to what each _Test_ is about?

Reply:
I agree @memodi . Its my feeling that Test 1, 2, and 3 and S, M and L are both generic naming conventions. But i don't think we can say node-density-heavy, ingress-perf, and cluster-density-v2. Would a customer have any idea what those are? I would guess no, but not sure if there's an alternative way to categorize these.

Reply:
I agree @memodi . Its my feeling that Test 1, 2, and 3 and S, M and L are both generic naming conventions. But i don't think we can say node-density-heavy, ingress-perf, and cluster-density-v2. Would a customer have any idea what those are? I would guess no, but not sure if there's an alternative way to categorize these.

Reply:
Whatever we decide here wrt naming conventions, we will need to apply it to the existing Resource consideration table updates as well.

Reply:
or we could add a footnote on each test, with a short description:

- Test 1 focuses on pods density per node, each of them generating a low traffic, on a small size cluster
- Test 2 focuses on ingress traffic throughput, on a medium size cluster
- Test 3 also focuses on pods density (like Test 1), running on larger cluster and generating more traffic

Reply:
cc @stleerh in case you want to chime in here

Reply:
short description as suggested [here](https://github.com/openshift/openshift-docs/pull/77127#discussion_r1657146887) sounds good to me. If we want to give specific metrics that represents scale (as OCP perf doc does) we collect `namespaceCount/podStatusCount/serviceCount` metrics for node/cluster-density workloads. For ingress-perf, we probably would need to rely on a metric that represents throughput of bytes (`nWorkloadBytesProcessedPerMinuteCadvisor`?).

Reply:
Hi @jotak, thanks for your input, I've updated the footnotes. In response to @memodi's comment here about adding metrics in footnotes, these would be good details to add but more certainty would be needed around the `ingress-perf` metrics. Just needed to clarify whether this is something you recommend adding to the existing docs or a detail for future reference? Thank you! 😊